conmon/conmon.c

```diff
@@ -1235,11 +1235,14 @@ int main(int argc, char *argv[])
 		exit(0);
 	}
 
+	if (opt_cid == NULL) {
+		fprintf(stderr, "Container ID not provided. Use --cid");
```

Comment:
With the direct `fprintf`, you need to explicitly add a trailing `\n`.

Reply:
Does this mean every caller to conmon now must exepcy the --cid option?

Reply:
> Does this mean every caller to conmon now must exepcy the --cid option?

Yup, but that's [not new][1].

[1]: https://github.com/kubernetes-incubator/cri-o/pull/1475/files#diff-caf7c5c19d8bb76309f55ccce30a55afL1241

Reply:
Yeah, we always need a container id.

Reply:
Then why not make it an argument, rather then an option. Options should be Optional.  :^(